droidlet/dashboard/web/src/components/AnnotationComponents/ObjectAnnotation.jsx

```diff
@@ -70,9 +70,6 @@ class ObjectAnnotation extends React.Component {
     if (!this.props.objects) {
       objects = this.props.stateManager.curFeedState.objects;
     }
-    if (objects === null) {
```

Comment:
i think you shouldn't remove this line of code right?

Reply:
I initially added the objects === null checker for the case for the camera, when there are no objects on the screen (and is null). This check is a bit wider than I intended to be (so I changed it just to be like this for the camera)